AcsfToolsCommands.php

```diff
@@ -192,6 +196,12 @@ public function ml($cmd, $command_args = '', $command_options = '', $options = [
 
         $this->output()->writeln("\n=> Running command on $domain");
         drush_invoke_process('@self', $cmd, $command_args, $drush_command_options);
+
```

Comment:
``````
      $i = 0;
      $delay = abs($options['delay']));
      foreach ($sites as $details) {
...
        drush_invoke_process('@self', $cmd, $command_args, $drush_command_options);

        // Delay in running the command for next site.
        if ($delay > 0 && $i < (count($sites) - 1)) {
          $this->output()->writeln("\n=> Sleeping for $delay seconds before running command on next site.");
          sleep($delay);
        }
      }

Reply:
❓ why do we need abs() here? also shouldn't we have isset check as it is optional param?

suggestion: we can avoid count($sites) call every-time by doing it outside

I know this is pseudo code but $i is not updated anywhere, can we use index of $sites array?

`foreach ($sites as $i => $details) {`